site/content/docs/user/using-wsl2.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Using WSL2"
+menu:
+  main:
+    parent: "user"
+    identifier: "using-wsl2"
+    weight: 1
+---
+# Using WSL2
+
+KIND can run using Windows Subsystem for Linux 2 (WSL2) on Windows 10 Insider builds. All the tools needed to build or run KIND work in WSL2, but some extra steps are needed to switch to WSL2. This page covers these steps in brief but also links to the official documentation if you would like more details.
```

Comment:
I remember some discussion about how to write KIND, cc: @BenTheElder

Reply:
I've given up on this :)

Reply:
The rest of the docs are lowercase "kind" so I updated to match